db/migrate/20180508142711_remove_invite_passthrough_hours.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveInvitePassthroughHours < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
@techAPJ This should be `up` only as you can't rollback the SQL statement.

Reply:
Good catch. Fixed in: https://github.com/discourse/discourse/commit/27efa4580038617c1a043aca17df8d2ca09ba746.